package/package.json

```diff
@@ -49,4 +49,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
+}
```

Comment:
Lots of missing trailing newlines. Doesn't help with the issue but need sorting.